README.md

```diff
@@ -1,26 +1,80 @@
+# Network-Viewer
 
-## Available Scripts
+## Introduction
+Network-Viewer is a **Redefined** HAR viewer human friendly view of requests which is build with [React](https://facebook.github.io/react/).
 
-In the project directory, you can run:
+The main purpose of this web application is to help you to view HAR files with many features like filters, search, timline charts etc.
 
-### `npm start`
+### Demo
+[Example](https://saucelabs.github.io/network-viewer/?file=https%3A%2F%2Fraw.githubusercontent.com%2Fsaucelabs%2Fnetwork-viewer%2Fmaster%2Fexamples%2Fsrc%2Fdata%2Fnetwork.har&isCORSEnabled=false)
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+![](https://i.imgur.com/wryjs8l.png)
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+## Features
+Main features of Network-Viewer are:
+- **Import/Upload** HAR file
+- **Fetch** HAR file using queryString url use `file=HAR_FILE_URL` and to enable CORS pass `isCORSEnabled=true` (by default CORS is disabled)
+- **Search** request by URL or body content
+- Filter request by **XHR, JS, CSS, Image, Media, Font, Doc, WebSocket, Manifest**
+- **Errors Only** filter to filter request by error status (where status code is greater than or equal to 400)
+- **WaterFall** chart to display timeline of request which includes 
+- **Stats** at footer to show number of requests, data transfered size, resoruces size, and time metrics like Page Load, DOMContentLoaded and Finished time
+- Request Detail section is available on click of request, response content is available for JSON and XML responses.
 
-### `npm test`
+## How to use
+### import/Upload HAR file
+Visit https://saucelabs.github.io/network-viewer/ and Import HAR file
 
-Launches the test runner in the interactive watch mode.<br />
+### Fetch HAR file
+- `file=HAR_FILE_URL` as queryString to pass HAR file url
+- `isCORSEnabled=true`as queryString to enable CORS support (by default CORS is disabled)
 
-### `npm build`
+### Use it as react library
+```
+$ npm install network-viewer
+```
 
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
+Example
+```
+import { NetworkViewer } from 'network-viewer';
+<NetworkViewer />
+```
 
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+#### Props
+| prop   | Type   | Default    | Values    | Description   |
+|:---|:---|:---|:---|:---|
+| autoHighlightChange   | Bool   | false   | true, false   | To allow change request highlight on scrollTimeStamp change when requestDetail is visible  |
+| containerClassName   | String   | null   |   | custom-class to overwrite network-viewer's default style  |
+| data   | Object   | null   | { log: { entries: [], pages: [] } }  | HAR file data which should contain log.entries and log.pages  |
```

Comment:
How do `log.entries` and `log.pages` elements look like?

Reply:
That is har file data, check this [har file](https://raw.githubusercontent.com/saucelabs/network-viewer/master/examples/src/data/network.har)

Incase someone modifies the HAR file, at least it should have entries and pages data.